src/app/public/modules/status-indicator/status-indicator.component.html

```diff
@@ -0,0 +1,42 @@
+<div *ngIf="descriptionType"
+  class="sky-status-indicator"
+>
+  <div
+    aria-hidden="true"
+    class="sky-status-indicator-icon"
+    [ngClass]="'sky-status-indicator-icon-' + indicatorType"
+  >
+    <span
+      class="sky-status-indicator-icon-default"
+    >
+      <sky-icon
+        [fixedWidth]="true"
+        [icon]="icon"
+      ></sky-icon>
+    </span>
+    <span
+      class="sky-status-indicator-icon-modern sky-margin-inline-xs"
+    >
+      <sky-icon-stack
+        size="xs"
+        [baseIcon]="baseIcon"
+        [topIcon]="topIcon"
+      >
+      </sky-icon-stack>
+    </span>
+  </div>
+  <div
+    class="sky-status-indicator-message-wrapper"
+  >
+    <span
+      class="sky-screen-reader-only"
```

Comment:
This `<span>` still shows up even if we set the `descriptionType` to `none`, just creating an empty element in the DOM. Should we throw an `*ngIf` on here to only make it appear when needed?